### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by berkatik**

### CODE REVIEW

**Comment by salotz** [comment]
@berkatik You must remove all of the non-code files from this MR. You should set your DS STORE to be ignored globally e.g. https://stackoverflow.com/questions/7335420/global-git-ignore#7335487

**Review by salotz — CHANGES_REQUESTED** [review]
Remove junk files. Impossible to understand the changes without it.

**Comment by berkatik** [comment]
@salotz I removed the .ds_store and similar files from the commit. Sphinx/src/_source has the files for compiling, should I get rid of them?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.DS_Store
+.buildinfo
+.git
+.nojekyll
+objects.inv
+.vscode/
```
Comment (salotz) [inline .gitignore:6]: The project gitignore should only contain ignores relevant to the project itself so you shouldn't need to add the following:

- `.vscode/`
- `.DS_Store`

These should be ignored in your global gitignore configuration.

Comment (berkatik) [reply .gitignore]: Ok, sorry. Updated the .gitignore accordingly.

**Comment by salotz** [comment]
@berkatik You need to rebase your commits so that no commit has the DS_Store files btw.
---

I was going to ask what you were trying to accomplish, but I see now that you are merging into the pages branch.

You shouldn't need to manually merge into the documentation branch, that should be automatically committed to with the tooling. The old instructions are here: https://github.com/ADicksonLab/wepy/blob/master/info/dev_guide.org#building-documentation

The relevant command to run is here if you want to do it manually: https://github.com/ADicksonLab/wepy/blob/44dec1b15ef45e88a3e056a28757681fa24fbcf6/tasks/modules/py.py#L354-L359

Overall you should make an MR with only the changes to the sources of the documentation. Then you just need to run the commands to build and deploy the docs to that branch.

File: .gitignore (hunk 1 above)
Comment (salotz) [reply .gitignore]: `.git` should not need to go into the `.gitignore` either. Git knows to ignore itself.

What are the other things in this listing you added?

**Comment by alexrd** [comment]
Yes, I was also thinking that having a way of sharing the changes to the docs prior to publishing would be ideal.  I see that we can just make changes to the org files in the master branch and make a PR for those.  @berkatik will do that instead :)  

Thanks for your help @salotz !

**Comment by salotz** [comment]
We can make this more automated as well by using Github Actions so that you only should need to commit the changes to the source files on `main` and the the pipeline can publish them. E.g. https://github.com/ADicksonLab/wepy/issues/79

**Pull request closed without merging**